JavaFiddle-ejb/src/main/java/ru/javafiddle/execution/test/CompileAndRunTest.java

```diff
@@ -0,0 +1,52 @@
+package ru.javafiddle.execution.test;
+
+import org.junit.Test;
+import ru.javafiddle.core.ejb.CompileAndRunBean;
+
+import static org.junit.Assert.fail;
+
+/**
+ * Created by nk on 08.04.2016.
+ */
+public class CompileAndRunTest {
+
+    @Test
+    public void CompileAndRunTest() {
+        CompileAndRunBean compileAndRunBean = null;
+        try {
+            compileAndRunBean = new CompileAndRunBean();
```

Comment:
Have you ever tried that working? 
You cannot initialize EJB (persistence manager) this way. In case your EJB code is dealing with DB, it won't work.